make/test.mk

```diff
@@ -76,6 +76,22 @@ e2e: $(BINDIR)/scratch/kind-exists | $(NEEDS_KUBECTL) $(NEEDS_GINKGO)
 e2e-ci: e2e-setup-kind e2e-setup
 	make/e2e-ci.sh
 
+$(BINDIR)/test/e2e.test: FORCE | $(NEEDS_GINKGO) $(BINDIR)/test
+	$(GINKGO) build --tags e2e_test test/e2e
+	mv test/e2e/e2e.test $(BINDIR)/test/e2e.test
```

Comment:
I couldn't find a way to tell `ginkgo build` where to put the binary so I `mv` it instead. 
 * https://onsi.github.io/ginkgo/#precompiling-suites